shoop/core/models/_contacts.py

```diff
@@ -170,7 +207,8 @@ def delete(self, *args, **kwargs):
 
     @property
     def groups(self):
```

Comment:
Are you sure? I think in your examble groups works since it comes from ContactGroup relation.